experiments/openebs/openebs-pool-disk-loss/vars.yml

```diff
@@ -0,0 +1,18 @@
+a_label: "{{ lookup('env','APP_LABEL') }}"
+a_ns: "{{ lookup('env','APP_NAMESPACE') }}"
+a_pvc: "{{ lookup('env','APP_PVC') }}"
+c_duration: "{{ lookup('env','TOTAL_CHAOS_DURATION') }}"
+c_experiment: "openebs-pool-disk-loss"
+ramp_time: "{{ lookup('env','RAMP_TIME') }}"
+data_persistence: "{{ lookup('env','DATA_PERSISTENCE') }}"
+liveness_label: "{{ lookup('env','LIVENESS_APP_LABEL') }}"
+liveness_namespace: "{{ lookup('env','LIVENESS_APP_NAMESPACE') }}"
+openebs_ns: "{{ lookup('env','OPENEBS_NAMESPACE') }}"
+cloud_platform: "{{ lookup('env','CLOUD_PLATFORM') }}"
+c_ns: "{{ lookup('env','CHAOS_NAMESPACE') }}"
+disk_name: "{{ lookup('env','DISK_NAME') }}"
+device_name: "{{ lookup('env', 'DEVICE_NAME') }}"
+node_name: "{{ lookup('env','NODE_NAME') }}"
+project_id: "{{ lookup('env','PROJECT_ID') }}"
+zone_name: "{{ lookup('env','ZONE_NAME') }}"
+auxiliary_appinfo: "{{ lookup('env','AUXILIARY_APPINFO') }}"
```

Comment:
please add c_engine var also

Reply:
Are we separating vars in a different file? If yes we need to update all the experiments

Reply:
It's okay to be in a seperate file. Every exp has different approach and style. Putting all the vars in a file looks messy.

Reply:
cc @ksatchit